repository/query/scripts/list_invitations_for_project.sql

```diff
@@ -0,0 +1,3 @@
+SELECT *
+FROM project_invitations
+WHERE project_id = @projectID
```

Comment:
Here, and in other listing methods, consider adding a sorting criterion (most simple case - `created_at`) so that it is always deterministic.